app/jobs/irs_reports/fraud_metrics_report.rb

```diff
@@ -0,0 +1,105 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'irs_reporting/fraud_metrics_lg99_report'
+
+module IrsReports
+  class FraudMetricsReport < Reports::BaseReport
+    REPORT_NAME = 'fraud-metrics-report'
+
+    attr_reader :report_date
+
+    def initialize(report_date = nil, *args, **rest)
+      @report_date = report_date
+      super(*args, **rest)
+    end
+
+    def perform(date = Time.zone.yesterday.end_of_day)
+      @report_date = date
+
+      report_configs.each do |config|
```

Comment:
```suggestion
      IdentityConfig.store.monthly_fraud_metrics_report_config.each do |config|
```

It looks like this method is only called once here, might be simpler to delete it and remove the abstraction.